src/common/constants.js

```diff
@@ -28,6 +28,7 @@ export const affiliate_signup_url = `https://login.${domain_url}/signup.php`
 export const affiliate_signin_url = `https://login.${domain_url}/signin.php`
 export const community_url = `https://community.${deriv_com_url}`
 export const oauth_url = `https://oauth.${domain_url}`
+export const terms_and_conditions_url = `https://${domain_url}`
```

Comment:
```suggestion
export const terms_and_conditions_url = `https://${domain_url}/terms-and-conditions/`
```
I'm not sure but according to the name of this constant, isn't this how it should be?

Reply:
i'll change the name of the constant